pkg/codegen/internal/test/testdata/output-funcs/dotnet/funcWithAllOptionalInputs.cs

```diff
@@ -0,0 +1,74 @@
+// *** WARNING: this file was generated by . ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.MadeupPackage.Codegentest
+{
+    public static class FuncWithAllOptionalInputs
+    {
+        /// <summary>
+        /// Check codegen of functions with all optional inputs.
+        /// </summary>
+        public static Task<FuncWithAllOptionalInputsResult> InvokeAsync(FuncWithAllOptionalInputsArgs? args = null, InvokeOptions? options = null)
+            => Pulumi.Deployment.Instance.InvokeAsync<FuncWithAllOptionalInputsResult>("madeup-package:codegentest:funcWithAllOptionalInputs", args ?? new FuncWithAllOptionalInputsArgs(), options.WithVersion());
+
+        public static Output<FuncWithAllOptionalInputsResult> Apply(FuncWithAllOptionalInputsApplyArgs? args = null, InvokeOptions? options = null)
```

Comment:
I suppose this should be called `Invoke`

Reply:
Good idea, implemented.